diff --git a/node_modules/@polkadot/api-augment/index.d.ts b/node_modules/@polkadot/api-augment/index.d.ts
index ae2e74e..ebae18c 100644
--- a/node_modules/@polkadot/api-augment/index.d.ts
+++ b/node_modules/@polkadot/api-augment/index.d.ts
@@ -1,2 +1 @@
 import './detectPackage.js';
\ No newline at end of file
-export * from './bundle.js';
diff --git a/node_modules/@polkadot/api-augment/index.js b/node_modules/@polkadot/api-augment/index.js
index ae2e74e..ebae18c 100644
--- a/node_modules/@polkadot/api-augment/index.js
+++ b/node_modules/@polkadot/api-augment/index.js
@@ -1,2 +1 @@
 import './detectPackage.js';
\ No newline at end of file
-export * from './bundle.js';
